app/utils/chain.ts

```diff
@@ -176,3 +177,61 @@ export const parseAssetString = (asset: string): [string, string] => {
 export const isEVMAddress = (address: string): boolean => {
   return address.startsWith("0x") && address.length === 42;
 };
+
+export const getChainKeyFromChain = (chain: string): ChainKey => {
+  chain = chain.toUpperCase();
+  switch (chain) {
+    case "AVAX": {
+      return ChainKey.AVALANCHE;
+    }
+    case "BSC": {
+      return ChainKey.BSCCHAIN;
+    }
+    case "ETH": {
+      return ChainKey.ETHEREUM;
+    }
+    case "BTC": {
+      return ChainKey.BITCOIN;
+    }
+    case "DOGE": {
+      return ChainKey.DOGECOIN;
+    }
+    case "LTC": {
+      return ChainKey.LITECOIN;
+    }
+    case "GAIA": {
+      return ChainKey.GAIACHAIN;
+    }
+    case "BCH": {
+      return ChainKey.BITCOINCASH;
+    }
+    case "THOR": {
+      return ChainKey.THORCHAIN;
+    }
+    default: {
+      return ChainKey.ETHEREUM;
+    }
+  }
+};
+
+export const getProviderTypeFromChain = (chain: string): ProviderKey => {
```

Comment:
Same here

Reply:
You are right but let's revisit on https://linear.app/project-chaos/issue/YLD-141/consolidate-all-chain-configuration because it's an impactful change